interfaces/http/routes/v1/assignments/list.go

```
@@ -36,14 +57,49 @@ func (h *httpHandler) list(ctx *fiber.Ctx) error {
 	var planId *uuid.UUID
 	var planassignments *pagination.PaginationView[models.PlanAssignment]
-	validFrom := ctx.Query("validFrom")
-	validUntil := ctx.Query("validUntil")
-	planIdOrSlug := ctx.Query("planIdOrSlug")
+	validFrom := ctx.Query("valid_from")
+	validUntil := ctx.Query("valid_until")
+	planIdOrSlug := ctx.Query("plan_id_or_slug")
 
 	// Create pagination input
 	paginationInput := pagination.ExtractPaginationFromContext(ctx)
 	c := context.WithValue(ctx.UserContext(), constants.TenantSlugKey, tenantSlug)
```

golangci-lint (1.64.8)

SA1029: should not use built-in type string as key for value; define your own type to avoid collisions

(staticcheck)